pkg/logfilter/logfilter.go

```diff
@@ -99,13 +111,27 @@ func GetLogFilters(processName string, messageTag string, ptpSettings map[string
 	messageTag = strings.ReplaceAll(messageTag, "{level}", "6")
 
 	if filter, ok := ptpSettings["stdoutFilter"]; ok {
-		logFilters = append(logFilters, logFilterFromRegex(filter, nil, "", "", nil)) // Filter anything with specified filter
+		logFilters = append(logFilters, logFilterFromRegex(filter, nil, "", "", nil, defaultEnhancedFilterThreshold)) // Filter anything with specified filter
 	}
 	if logReduce, ok := ptpSettings["logReduce"]; ok {
-		if strings.ToLower(logReduce) == "true" || strings.ToLower(logReduce) == "basic" {
-			logFilters = append(logFilters, logFilterFromRegex("^.*master offset.*$", nil, "", "", nil)) // Just filter anything with master offset
-		} else if strings.ToLower(logReduce) == "enhanced" {
-			logFilters = getLogFiltersEnhanced(processName, messageTag, defaultEnhancedFilterTime)
+		logReduceSettings := strings.Split(strings.TrimSpace(logReduce), " ")
```

Comment:
string.Fields will do the same without having to specify the space and will handle multiple spacing making it a bit more robust